src/app/index.ts

```diff
@@ -274,25 +271,17 @@ module.exports = yo.extend({
 
         // Copy project template files from project repository (currently only custom functions has its own separate repo)
         if (projectRepoBranchInfo.repo) {
-          git().clone(projectRepoBranchInfo.repo, this.destinationPath(), ['--branch', projectRepoBranchInfo.branch || 'master'], async (err) => {
-            // for all project types other than Excel Custom Functions. modify the generated project so it targets the selected host
-            if (!this.project.isExcelFunctionsProject) {
-              // Call 'convert-to-single-host' npm script in generated project, passing in host parameter
-              const cmdLine = `npm run convert-to-single-host --if-present -- ${_.toLower(this.project.hostInternalName)}`;
-              await childProcessExec(cmdLine); 
-            }
-            
-            // modify manifest guid and DisplayName
-            await modifyManifestFile(`${this.destinationPath()}/manifest.xml`, 'random', `${this.project.name}`);
-            
-            // delete the .git folder after cloning over repo
-            const gitFolder = path.join(this.destinationPath(), '.git');
-            if (fs.existsSync(gitFolder)) {
-              helperMethods.deleteFolderRecursively(gitFolder);
-            }
-
-            return err ? reject(err) : resolve();
-          });
+          await helperMethods.downloadProjectTemplate(this.destinationPath(), projectRepoBranchInfo.repo, projectRepoBranchInfo.branch);
+              
+          // modify manifest guid and DisplayName
+          await modifyManifestFile(`${this.destinationPath()}/manifest.xml`, 'random', `${this.project.name}`);
+
+          if (!this.project.isExcelFunctionsProject) {
```

Comment:
don't need to hard-code for isExcelFunctionsProject since the --if-present flag allows it to be a no-op if there is no convert-to-single-host script. I missed this with previous code review.